src/_pytest/faulthandler.py

```diff
@@ -72,14 +79,19 @@ def get_timeout_config_value(config: Config) -> float:
     return float(config.getini("faulthandler_timeout") or 0.0)
 
 
+def get_exit_on_timeout_config_value(config: Config) -> bool:
+    return bool(config.getini("faulthandler_exit_on_timeout"))
```

Comment:
Note to reviewers: the `bool()` cast is needed to please mypy but `getini` itself ensures that it is a boolean value at runtime because of the `type="bool"` kwarg passed to `addini` but the type checkers are probably blind to this.

Alternatively, I could have written:

```suggestion
    return config.getini("faulthandler_exit_on_timeout") is True
```

which is safer to avoid a false positive cast to `True` in case getini does not provide a bool value but I found it uglier.

Let me know what you prefer.

Reply:
That would be useful as a code comment. Additionally, if it's just for MyPy, it might make sense to either use a `typing.cast()` or (probably better) add an `assert` statement telling MyPy to perform type narrowing.